src/editors/sharedComponents/TinyMceWidget/customTinyMcePlugins/embedIframePlugin.test.js

```diff
@@ -162,4 +161,248 @@ describe('TinyMCE Embed IFrame Plugin', () => {
       }),
     );
   });
+  it('tests onChange function in plugin', () => {
```

Comment:
can we specify that we're only testing the iframe plugin?

Reply:
Added